client/state/plans/get-interval-type-for-term.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from 'lib/plans/constants';
+
+export default function getIntervalTypeFromTerm( term ) {
+	return (
+		{
+			[ TERM_MONTHLY ]: 'monthly',
+			[ TERM_ANNUALLY ]: 'yearly',
+			[ TERM_BIENNIALLY ]: '2yearly',
+		}[ term ] || null
+	);
```

Comment:
Are the parens necessary?

Reply:
Don't question `prettier`!

The line break after return makes them necessary 🙂